packages/core/src/domain/tracekit/tracekit.ts

```diff
@@ -67,7 +67,7 @@ function instrumentOnError(callback: UnhandledErrorCallback) {
         message,
         stack,
       }
-      callback(stackTrace, messageObj)
+      callback(stackTrace, errorObj ?? messageObj)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏this could be refactored like this:

```ts
let stackTrace: StackTrace
if (errorObj instanceof Error) {
     stackTrace = computeStackTrace(errorObj)
} else {
      const stack = [{ url, column, line }]
      const { name, message } = tryToParseMessage(messageObj)
      stackTrace = {
        name,
        message,
        stack,
      }
}
callback(stackTrace, errorObj ?? messageObj)
```